### PR TITLE
 ocd ids for '2019 Austrian legislative election'

### PR DESCRIPTION
**Pull request by sguenther85**

### CODE REVIEW

File: identifiers/country-at/districts.csv [hunk 1]
```diff
@@ -0,0 +1,39 @@
+ocd-division/country:at/state:burgenland/cd:burgenland_nord,Burgenland Nord
```
[the remark this answers is not in the document]
Comment (sguenther85) [reply identifiers/country-at/districts.csv]: ok, i replace the state-names with the two letter "hasc" code.

File: identifiers/country-at.csv [hunk 2]
```diff
@@ -0,0 +1 @@
...
ocd-division/country:at/state:vorarlberg/cd:vorarlberg_nord,Vorarlberg Nord
ocd-division/country:at/state:vorarlberg/cd:vorarlberg_süd,Vorarlberg Süd
ocd-division/country:at/state:wien,Wien
ocd-division/country:at/state:wien/cd:wien_innen-ost,Wien Innen-Ost
ocd-division/country:at/state:wien/cd:wien_innen-süd,Wien Innen-Süd
ocd-division/country:at/state:wien/cd:wien_innen-west,Wien Innen-West
ocd-division/country:at/state:wien/cd:wien_nord,Wien Nord
ocd-division/country:at/state:wien/cd:wien_nord-west,Wien Nord-West
ocd-division/country:at/state:wien/cd:wien_süd,Wien Süd
ocd-division/country:at/state:wien/cd:wien_süd-west,Wien Süd-West
```
[the remark this answers is not in the document]
Comment (sguenther85) [reply identifiers/country-at.csv]: yes, I saw it too, although it is still displayed correctly.
I check it again

File: identifiers/country-at/districts.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (sguenther85) [reply identifiers/country-at/districts.csv]: done

File: identifiers/country-at.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (sguenther85) [reply identifiers/country-at.csv]: should looking good now

**Comment by sguenther85** [comment]
changes implemented

**Comment by jdmgoogle** [comment]
Thanks. Merging.

**Pull request merged**